genome_designer/millstone_setup.py

```diff
@@ -116,7 +116,14 @@
             'https://www.dropbox.com/s/y83fqsdq73pt1w8/velvet-linux.zip'
         ]
     },
-
+    'seqan': {
+        'Linux': [
+            'https://www.dropbox.com/s/4vel0v42cmh5nqk/place_contig'
+        ],
+        'Darwin': [
+            'https://www.dropbox.com/s/5q49fy7w1j5u4bg/place_contig'
+        ]
+    }
 }
 
```

Comment:
Need to update `TOOLS_TO_EXECUTABLES` below with
    ...
    'seqan': ['place_contig'],
    ...